src/Discord.Net.Core/Audio/IAudioClient.cs

```diff
@@ -20,6 +21,9 @@ public interface IAudioClient : IDisposable
         /// <summary> Gets the estimated round-trip latency, in milliseconds, to the voice UDP server. </summary>
         int UdpLatency { get; }
 
+        /// <summary>Gets the current audio streams.</summary>
+        Dictionary<ulong, AudioInStream> GetStreams();
```

Comment:
It should be a read-only dictionary so users don't try to add or remove from it.
I would suggest changing it to `IReadOnlyDictionary`.